core/src/sql/statements/define/field.rs

```diff
@@ -225,7 +225,7 @@ impl Display for DefineFieldStatement {
 			f.write_char(' ')?;
 			None
 		};
-		write!(f, "{}", self.permissions)?;
+		write!(f, "{:#}", self.permissions)?;
```

Comment:
Does the pretty print return something different that no pretty?
Pretty should be only about formatting

Reply:
I documented it [here](https://github.com/surrealdb/surrealdb/pull/4887/files#diff-eaf84239692fd3a3835c84b8fac49145103191d5ec882af39bc36b25e3720e5fR76-R81). According to the Rust documentation this represents an [alternate formatting](https://doc.rust-lang.org/std/fmt/#sign0) of the same data. Since we only have two ways of displaying permissions (for tables and fields), this seemed like the cleaner solution, as `fmt` cannot take any arguments and this is just a temporary solution before we can make a breaking change.

Reply:
I have documented it on the same line in https://github.com/surrealdb/surrealdb/pull/4887/commits/7ea4263055717dd161dc74524eec8f8ab7fe8f6c.